egs2/dns_ins21/enh1/local/data.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+help_message=$(cat << EOF
+Usage: $0 [--stage <stage>] [--stop_stage <stop_stage>] [--configure <conf.json>]
+  optional argument:
+    [--stage]: 1 (default) or 2
+    [--stop_stage]: 1 or 2 (default)
+    [--configure]: use another specific configuration file
```

Comment:
This option is used in the recipe for the previous DNS challenge. But, I decided to remove the configure option because the config is updated in `dns_create_mixture.sh`.

Reply:
And, the config file should not be changed to reproduce the dataset of the challenge.